app/Models/Plan/PlanDay.php

```diff
@@ -135,11 +135,10 @@ public function complete(int $user_id = null) : void
         if (is_null($user_id)) {
             $user_id = Auth::user()->id;
         }
-        $completed_item = PlanDayComplete::firstOrNew([
+        PlanDayComplete::updateOrCreate([
```

Comment:
is this the same as upsert? if so, name it upsert

Reply:
I have change it to use upsert instead of updateOrCreate. See: https://github.com/faithcomesbyhearing/dbp/pull/937/commits/50131cd2aaa5ca4a38819adf4bd5dbf8f8ecf668#diff-9177aefcb0bcc4f977c5cf6255012ba7b96ed2ab2d359c38c8ef1b658e611207R143